Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -66,9 +66,7 @@ class BrowserViewController: UIViewController {
     private var customSearchBarButton: UIBarButtonItem?
     var openedUrlFromExternalSource = false
     var passBookHelper: OpenPassBookHelper?
-    // TODO: Remove at FXIOS-5639 Feature flag like to use during the refactoring
-    private var shouldUseOverlayManager = false
-    private var overlayManager: OverlayModeManager?
+    var overlayManager: OverlayModeManager!
```

Comment:
Do we absolutely need the `!` or can it be `?`

Reply:
The problem is that it needs the URLBar to initialize which is done in viewDidLoad, I will initialize in BVC init and set the urlbar after when is available